app/components/ui/Carousel.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const CarouselOuter = styled.div`
+  position: relative;
+  overflow: visible;
+  width: 100%;
+`;
+
+const CarouselContainer = styled.div`
+  overflow: hidden;
+`;
+
+const CarouselSlider = styled.div`
+  position: relative;
+  display: flex;
+  flex-wrap: nowrap;
+  transition: transform 1s ease;
+  transform: translateX(${props => -props.activeIndex * props.slotWidth}%);
+`;
+
+const CarouselNavButton = ({ dir, onClick }) => (
+  <button type="button" className={`carousel-nav btn-${dir}`} onClick={onClick} />
+);
+CarouselNavButton.props = {
+  dir: PropTypes.string.isRequired,
+};
+
+const CarouselSlot = styled.div`
+  flex: 1 0 calc(${props => props.width}% - 10px);
+  margin-right: 10px;
+`;
+
+class Carousel extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeIndex: 0,
+    };
+  }
+
+  render() {
+    const advanceRight = () => {
+      const { activeIndex } = this.state;
+      const { children, numSlots } = this.props;
+      this.setState({
+        activeIndex: Math.min(activeIndex + 1, children.length - numSlots),
+      });
+    };
+
+    const advanceLeft = () => {
+      const { activeIndex } = this.state;
+      this.setState({ activeIndex: Math.max(0, activeIndex - 1) });
+    };
+
+    const { activeIndex } = this.state;
+    const { children, numSlots } = this.props;
+    const slotWidth = 100 / numSlots;
```

Comment:
Something's a bit odd with this when viewed on mobile. I can't actually scroll to the right far enough to see all of the items:

<img width="471" alt="Screen Shot 2019-07-21 at 7 44 00 PM" src="https://user-images.githubusercontent.com/1002748/61602808-f22bc900-abef-11e9-9e59-a5581ae4eb49.png">